spec/bns/fetcher/notion/birthday_next_week_spec.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+RSpec.describe Fetcher::Notion::BirthdayNextWeek do
+  before do
+    @config = {
+      database_id: "5aebbdc871864f4fbab5596a41a81a01",
+      secret: "secret_K5UCqm27GvAscTlaGJmS2se4fyM1K7is3OIZMw03NaC"
+    }
+
+    @fetcher = described_class.new(@config)
+  end
+
+  describe "attributes and arguments" do
+    it { expect(@fetcher).to respond_to(:config) }
+
+    it { expect(described_class).to respond_to(:new).with(1).arguments }
+    it { expect(@fetcher).to respond_to(:fetch).with(0).arguments }
+  end
+
+  describe ".fetch" do
+    it "fetch data from the given configured notion database" do
```

Comment:
I see the coveralls remains in 100%, however, do we need to cover the fetching with a filter, like the other use cases?

Reply:
Hello @L-Zuluaga, yes, you are right. In this case, we are using the filter defined in the `Fetcher::Notion::BirthdayNextWeek` module. The filter can not be blank, since we are defining it directly in the fetcher module, but I didn't update the name of the VCR file `"/notion/birthdays_next_week/fetch_without_filter"`. I'm going to update it. Thanks! :+1: